packages/nimble-components/src/patterns/error/specs/checkbox-radio-error.md

```diff
@@ -0,0 +1,65 @@
+# Error State on Checkbox and Radio Group
```

Comment:
Note: I wasn't sure the best place to put this HLD since is spanned 2 components. Therefore, I put it in the error pattern directory. I'm open to alternatives.

Reply:
I had similar uncertainty about where to put the "required input" HLD, as it also affects multiple components, and the pattern would potentially apply to future components in `spright-components`, too. I started my HLD at `nimble/specs/required-inputs/README.md`, but I'm not sure if that's a better place than you've chosen. (The only other (non-template) specs there are for `labels-and-localization` and `nimble-blazor`.)

Reply:
I think both locations are sane and in line with our guidance and don't have a strong preference between them so I'm fine to leave each PR as-is.

Reply:
This is no longer applicable. There are now 2 separate HLDs in this PR.